app/bundles/ReportBundle/Tests/Model/ExportHandlerTest.php

```diff
@@ -36,10 +36,10 @@ public function testHandler()
 
         $this->assertTrue(is_resource($handler));
 
-        $handler = $exportHandler->closeHandler($handler);
+        $exportHandler->closeHandler($handler);
 
         $this->assertFalse(is_resource($handler));
-        $this->assertNull($handler);
+        $this->assertNotNull($handler);
```

Comment:
This is always a resource, as the handler was never returned from the `closeHandler()` method.

See:

![Uploading Screenshot from 2023-12-05 15-19-40.png…]()